openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Lineage.spec.ts

```diff
@@ -1581,4 +1590,300 @@ test.describe.serial('Test pagination in column level lineage', () => {
+    const downstreamTable = new TableClass();
+
+    try {
+      // Create test entities
+      await Promise.all([
+        currentTable.create(apiContext),
+        upstreamTable.create(apiContext),
+        downstreamTable.create(apiContext),
+      ]);
+
+      await test.step('Create lineage connections', async () => {
+        const currentTableId = currentTable.entityResponseData?.id;
+        const upstreamTableId = upstreamTable.entityResponseData?.id;
+        const downstreamTableId = downstreamTable.entityResponseData?.id;
+
+        if (!currentTableId || !upstreamTableId || !downstreamTableId) {
+          throw new Error('Entity IDs are missing');
+        }
+
+        await connectEdgeBetweenNodesViaAPI(
+          apiContext,
+          {
+            id: upstreamTableId,
+            type: 'table',
+          },
+          {
+            id: currentTableId,
+            type: 'table',
+          },
+          []
+        );
+        await connectEdgeBetweenNodesViaAPI(
+          apiContext,
+          {
+            id: currentTableId,
+            type: 'table',
+          },
+          {
+            id: downstreamTableId,
+            type: 'table',
+          },
+          []
+        );
+      });
+
+      await test.step('Navigate to lineage tab and verify custom properties tab in sidebar', async () => {
+        // Navigate to the entity detail page first (required for visitLineageTab)
+        const searchTerm =
+          currentTable.entityResponseData?.['fullyQualifiedName'] ||
+          currentTable.entity.name;
+        if (!searchTerm) {
+          throw new Error('Cannot determine search term for entity');
+        }
+        await currentTable.visitEntityPage(page, searchTerm);
+
+        // Navigate to lineage tab (this navigates to the full lineage page)
+        await visitLineageTab(page);
+
+        // Click on the current entity node to open the sidebar drawer
+        const nodeFqn =
+          currentTable.entityResponseData?.['fullyQualifiedName'];
+        if (!nodeFqn) {
+          throw new Error('Entity fullyQualifiedName is missing');
+        }
+        await clickLineageNode(page, nodeFqn);
+
+        // Wait for the lineage entity panel (sidebar drawer) to open
+        const lineagePanel = page.getByTestId('lineage-entity-panel');
+        await expect(lineagePanel).toBeVisible();
+
+        // Wait for the panel content to load
+        await waitForAllLoadersToDisappear(page);
+
+        // Try to find custom properties tab in the lineage sidebar - use data-testid first (priority 1)
+        let customPropertiesTab = lineagePanel.getByTestId('custom-properties-tab');
+        const isTabVisible = await customPropertiesTab.isVisible();
+
+        if (!isTabVisible) {
+          // Fallback: use getByRole (priority 2) within the lineage panel
+          customPropertiesTab = lineagePanel.getByRole('menuitem', {
+            name: /custom propert/i,
+          });
+        }
```

Comment:
Any reason we need fallback? we should relay on data-testid if its not available it means something wrong.